README.md

```diff
@@ -40,26 +40,26 @@ Here are some docs which we recommend you look at first when getting started wit
 
 Want to get your code to space? Become a contributor!
 
-Check out our doc on [contributing to KubOS](https://docs.kubos.com/latest/contributing/contribution-process.html) 
-and come talk to us on [Slack](https://slack.kubos.co/) to join our community. 
+Check out our doc on [contributing to KubOS](https://docs.kubos.com/latest/contributing/contribution-process.html)
+and come talk to us on [Discord](https://kubos-preservation-group.github.io/DiscordRedirect) to join our community.
 
-Or, if you're just looking to give some feedback, 
-submit an [issue](https://github.com/kubos/kubos/issues) with your feature requests or bug reports! 
+Or, if you're just looking to give some feedback,
+submit an [issue](https://github.com/kubos/kubos/issues) with your feature requests or bug reports!
 
 # Repo Components
 
 These are the important folders in this repo:
 
-- apis - Contains all of our [hardware APIs](https://docs.kubos.com/latest/deep-dive/apis/device-api/index.html)
-  as well as our [application APIs](https://docs.kubos.com/latest/os-docs/apps/app-guide.html)
-- clients - Contains client programs which can be used locally to communicate with the
+- APIs - Contains all of our [hardware APIs](https://docs.kubos.com/latest/deep-dive/apis/device-api/index.html)
+  as well as our [application APIs](https://docs.kubos.com/1.21.0/ecosystem/apps/app-guide.html#apis)
+- Clients - Contains client programs which can be used locally to communicate with the
   corresponding service on a remote target
-- docs - Contains all of the raw doc files used to generate our [documentation](http://docs.kubos.co)
-- examples - Various example KubOS projects
-- hal - A collection of [hardware interface abstractions](https://docs.kubos.com/latest/apis/kubos-hal/index.html)
-- libs - Helper libraries used when building [Kubos services](https://docs.kubos.com/latest/os-docs/index.html#services)
-- services - Contains all of the Kubos [core services](https://docs.kubos.com/latest/os-docs/services/core-services.html)
-  and [hardware services](https://docs.kubos.com/latest/os-docs/services/hardware-services.html)
+- Docs - Contains all of the raw doc files used to generate our [documentation](https://kubos-preservation-group.github.io/kubos/)
+- Examples - Various example KubOS projects
+- Hal - A collection of [hardware interface abstractions](https://docs.kubos.com/1.21.0/deep-dive/apis/kubos-hal/index.html)
+- Libs - Helper libraries used when building [Kubos services](https://docs.kubos.com/1.21.0/deep-dive/apis/kubos-libs.html)
+- Services - Contains all of the Kubos [core services](https://docs.kubos.com/1.21.0/ecosystem/services/core-services.html)
+  and [hardware services](https://docs.kubos.com/1.21.0/ecosystem/services/hardware-services.html)
```

Comment:
It looks like this change also hardcoded the documentation links to 1.21.0. Could you change the links to docs.kubos.com back to "latest" so the links continue to point to the latest version of the docs?

Reply:
> It looks like this change also hardcoded the documentation links to 1.21.0. Could you change the links that point to these specific docs pages so that they point to the "latest" version instead of specifically 1.21.0?

I have worked on the other comment. But I don't know how to change the link.

Reply:
right now it looks like the links in this section (lines 59-62) of the README have `1.21.0` in them, when instead it should be `latest`. This is to prevent these links from pointing to old documentation if a `1.22.0` gets released. To use the hardware services link as an example, the link should look like this `[hardware services](https://docs.kubos.com/latest/ecosystem/services/hardware-services.html)`. 

Thanks again for your contributions! Let me know if you prefer for me to make this last fix

Reply:
Hi, sadly, I don't know how to change that. If you can work on this last fix, I'd appreciate it.